scripts/mocha.opts

```diff
@@ -1,3 +1,2 @@
 --compilers js:babel-register
---require jsdom-global/register
```

Comment:
Discovered that this led to not having the html page of jsdom cleaned between tests. Ugh